data-processing/tests/test_extract_definitions.py

```diff
@@ -85,6 +85,70 @@ def test_extract_nicknames_from_before_symbols():
     assert nickname1.definition_text == "timestep"
 
 
+def test_extract_nicknames_symbols_separated_by_colon():
+    text = "The agent acts with SYMBOL in each timestep SYMBOL, where SYMBOL : policy."
```

Comment:
Can this sentence be made even shorter, e.g., something like "We defined SYMBOL: policy."? A shorter sentence minimizes the things that could accidentally yield other nicknames as we extend the nickname extractor. For instance, we already check on the pattern "timestep SYMBOL" in another test, so including it here makes more work for us if we change the behavior of the nickname extractor later.